src/Microsoft.AspNet.Cors.Core/EnableCorsAttribute.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.AspNet.Cors.Core
+{
+    /// <inheritdoc />
+    [AttributeUsage(AttributeTargets.Class | AttributeTargets.Method, AllowMultiple = false, Inherited = false)]
```

Comment:
I am sorry my bad, i thought i had not marked it as inherited false, (hence the comment about default).. yes you are right, this will be fixed.